src/main/frontend/components/header.css

```diff
@@ -111,6 +110,7 @@
   max-width: 7ch;
   color: var(--ls-icon-color, #045591);
   height: 14px;
+  @apply md:max-w-none;
```

Comment:
This will override the `max-width: 7ch` above which is to limit the width for the repo name.
Shall we just remove this line?
![CleanShot 2021-08-12 at 11 56 36](https://user-images.githubusercontent.com/479169/129136163-ff875295-72e7-4d6f-8741-0741d32106c9.png)

Reply:
Sure, I moved this line because originally it is put at the top, which will be overridden in all cases.